lib/streamlit/DeltaGenerator.py

```diff
@@ -152,14 +166,14 @@ class NoValue(object):
 class DeltaGenerator(object):
     """Creator of Delta protobuf messages."""
 
-    def __init__(
-        self,
-        enqueue,
-        id=0,
-        is_root=True,
-        container=BlockPath_pb2.BlockPath.MAIN,
-        path=(),
-    ):
+    def __init__(self,
+                 enqueue,
+                 id=0,
+                 delta_type=None,
+                 last_index=None,
+                 is_root=True,
+                 container=BlockPath_pb2.BlockPath.MAIN,
+                 path=()):
```

Comment:
I know this is not your doing, but can you add documentation to all these parameters? For example:

```
enqueue: callable or None
  (keep existing text)
id: int or None
  (keep existing text)
delta_type: string or None
  The name of the element passed in Element.proto's oneof. This is needed so we can transform dataframes for some elements when performing an `add_rows`.
last_index: int or None
  The last index of the DataFrame for the element this DeltaGenerator created. Only applies to elements that transform dataframes, like line charts.
is_root: bool
  If True, this will behave like a root DeltaGenerator which an auto-incrementing ID (in which case, `id` should be None). If False, this will have a fixed ID as determined by the `id` argument.
container: BlockPath
  The root container for this DeltaGenerator. Can be MAIN or SIDEBAR.
path: tuple of ints
  The full path of this DeltaGenerator, consisting of the IDs of all ancestors. The 0th item is the topmost ancestor.
```